velox/core/QueryConfig.h

```diff
@@ -244,6 +244,11 @@ class QueryConfig {
   /// which uses ~128MB memory when to fill a spill run.
   static constexpr const char* kMaxSpillRunRows = "max_spill_run_rows";
 
+  /// The max bytes to fill for each query. This is used to cap the storage used
+  /// for spilling. If it is zero, then there is no limit and spilling might
+  /// exhaust the storage. The default value is set to 100 TB.
+  static constexpr const char* kMaxSpillBytes = "max_spill_bytes";
```

Comment:
I think Presto java has experimental.query-max-spill-per-node for this. Can you add to map this to max_spill_bytes once after this change is done?

Reply:
OK